src/main/java/nl/knaw/dans/ingest/core/service/DepositIngestTask.java

```diff
@@ -337,6 +337,10 @@ Dataset getMetadata() {
         var date = getDateOfDeposit();
         var contact = getDatasetContact();
         var deposit = getDeposit();
+        var accessibleToValues = XPathEvaluator
+            .strings(deposit.getFilesXml(), "//files:accessibleToRights")
```

Comment:
Use an absolute XPath if the element searched for can only possibly be at a specific level. In this case: 

"/files:files/files:file/files:accessibleToRights", I think.